api/nakadi-event-bus-api.yaml

```diff
@@ -1132,6 +1161,13 @@ definitions:
           If set by the producer Event will be rejected.
         format: date-time
         example: '1996-12-19T16:39:57-08:00'
+      version:
+        type: string
+        description: |
+          Version of the schema used for validating this event. This is enriched upon reception.
+          Clients consuming events should desserialize events only when differences between their
+          schema and the one used to validate differs on minor level, for example, from '1.1.0' to
+          '1.2.0'.
```

Comment:
Yes it is. Do you we might better make it explicit? I thought it would be implicit since PATCH is within MINOR.